rpms/build.sh

```diff
@@ -66,8 +66,9 @@ cp -R wazuh-* ${build_dir}/${package_name}
 
 # Including spec file
 if [ "${use_local_specs}" = "no" ]; then
-    curl -sL https://github.com/wazuh/wazuh-packages/tarball/${wazuh_packages_branch} | tar zx
-    specs_path=$(find ./wazuh* -type d -name "SPECS" -path "*rpms*")
+    specs_path="/d_specs"
+    mkdir -p "${specs_path}"
+    curl -L "https://raw.githubusercontent.com/wazuh/wazuh-packages/${wazuh_packages_branch}/rpms/SPECS/wazuh-${build_target}.spec" -o "/d_specs/wazuh-${build_target}.spec"
```

Comment:
Use ${specs_path} variable instead of hardcoded value "/d_specs/..."

Reply:
done [760d801](https://github.com/wazuh/wazuh-packages/pull/1571/commits/760d801a242f59dbd739a73b0bd66b8291aeaf66)